python-sdk/src/forta_agent/label.py

```diff
@@ -17,21 +18,30 @@ class LabelType(IntEnum):
     Contract = 12
     Good = 13
 
+
 class EntityType(IntEnum):
     Unknown = 0
     Address = 1
     Transaction = 2
     Block = 3
     Url = 4
 
+
 class Label:
     def __init__(self, dict):
         assert_enum_value_in_dict(dict, 'entity_type', EntityType)
         assert_enum_value_in_dict(dict, 'label_type', LabelType)
         assert_non_empty_string_in_dict(dict, 'entity')
         self.entity = dict['entity']
         self.confidence = dict['confidence']
-        self.customValue = dict['custom_value']
+        self.custom_value = dict.get('custom_value')
         self.entity_type = dict['entity_type']
         self.label_type = dict['label_type']
 
+    def toDict(self):
+        d = dict(self.__dict__, **{
+            'entityType': self.entity_type,
+            'labelType': self.label_type,
+            'customValue': self.custom_value
+        })
+        return {k:v for k,v in d.items() if v is not None}
```

Comment:
Is there a way to unit test that this will serialize correctly.

Reply:
since the printing to console currently happens in the nodejs process, there isn't a nice way to unit test. once the sdk is split properly (i.e. strictly using gRPC) this will probably not be necessary